components/expenses/ExpenseFormPayeeInviteNewStep.js

```diff
@@ -0,0 +1,321 @@
+  },
+  payoutOptionLabel: {
+    id: `ExpenseForm.PayoutOptionLabel`,
+    defaultMessage: 'Payout method',
+  },
+  invoiceInfo: {
+    id: 'ExpenseForm.InvoiceInfo',
+    defaultMessage: 'Additional invoice information',
+  },
+  invoiceInfoPlaceholder: {
+    id: 'ExpenseForm.InvoiceInfoPlaceholder',
+    defaultMessage: 'Tax ID, VAT number, etc. This information will be printed on your invoice.',
+  },
+  country: {
+    id: 'ExpenseForm.ChooseCountry',
+    defaultMessage: 'Choose country',
+  },
+  address: {
+    id: 'ExpenseForm.AddressLabel',
+    defaultMessage: 'Physical address',
+  },
+  recipientNoteLabel: {
+    id: 'ExpenseForm.RecipientNoteLabel',
+    defaultMessage: 'Add a note for the recipient',
+  },
+  additionalInfo: {
+    id: 'ExpenseForm.inviteAdditionalInfo',
+    defaultMessage: 'If you have additional information about the payment details, you can add them by clicking below.',
+  },
+  additionalInfoBtn: {
+    id: 'ExpenseForm.inviteAdditionalInfoBtn',
+    defaultMessage: '<Link>Add Additional Details</Link>',
+  },
+});
+
+const ExpenseFormPayeeInviteNewStep = ({ formik, collective, onCancel, onNext }) => {
+  const intl = useIntl();
+  const { formatMessage } = intl;
+  const { values, errors } = formik;
+  const stepOneCompleted = values.payee?.name && values.payee?.email;
+
+  const setPayoutMethod = React.useCallback(({ value }) => formik.setFieldValue('payoutMethod', value), []);
+  const [showAdditionalInfo, setAdditionalInfo] = React.useState(false);
+
+  return (
+    <Fragment>
+      <Grid
+        gridTemplateColumns={['100%', 'calc(50% - 8px) calc(50% - 8px)']}
+        gridColumnGap={[null, 2, null, 3]}
+        gridAutoFlow="dense"
+      >
+        <Box>
+          <Field name="payee.name">
+            {({ field }) => (
+              <StyledInputField name={field.name} label={formatMessage(msg.nameLabel)} labelFontSize="13px" mt={3}>
+                {inputProps => <StyledInput {...inputProps} {...field} />}
+              </StyledInputField>
+            )}
+          </Field>
+        </Box>
+        <Box>
+          <Field name="payee.email" required>
+            {({ field }) => (
+              <StyledInputField
+                name={field.name}
+                label={formatMessage(msg.emailTitle)}
+                labelFontSize="13px"
+                error={errors.payee?.email}
+                mt={3}
+              >
+                {inputProps => <StyledInput {...inputProps} {...field} type="email" />}
+              </StyledInputField>
+            )}
+          </Field>
+        </Box>
+
+        {!showAdditionalInfo ? (
+          <Box gridColumn={[null, '1 / span 2']} mt={3}>
+            <MessageBox type="info" fontSize="12px">
+              {formatMessage(msg.additionalInfo)}
+              <br />
+              {formatMessage(msg.additionalInfoBtn, {
+                Link: getI18nLink({ onClick: () => setAdditionalInfo(true) }),
+              })}
```

Comment:
:ice_cream: I don't see a strong value in having `<Link>` inside the message, it makes the template string more complex.

```suggestion
              <a href="#" onClick={() => setAdditionalInfo(true)}>
                {formatMessage(msg.additionalInfoBtn}
              </a>
```

Also, according to https://github.com/opencollective/opencollective/blob/master/rfcs/010-react-intl.md, we should try to prefer using `FormattedMessage` when possible.